tools/c7n_awscc/build.py

```diff
@@ -0,0 +1,132 @@
+    "applicationinsights": "application-insights",
+    "applicationautoscaling": "application-autoscaling",
+    "aps": "amp",
+    "backupgateway": "backup-gateway",
+    "cassandra": "keyspaces",
+    "certificatemanager": "acm",
+    "codestarconnections": "codestar-connections",
+    "codestarnotification": "codestar-notifications",
+    "cognito": "cognito-identity",
+    "customerprofiles": "customer-profiles",
+    "devopsguru": "devops-guru",
+    "elasticloadbalancingv2": "elbv2",
+    "elasticloadbalancing": "elb",
+    "emrserverless": "emr-serverless",
+    "emrcontainers": "emr-containers",
+    "eventschemas": "schemas",
+    "elasticsearch": "es",
+    "inspectorv2": "inspector2",
+    "iotcoredeviceadvisor": "iotdeviceadvisor",
+    "kinesisfirehose": "kinesis-firehose",
+    "lex": "lexv2-models",
+    "licensemanager": "license-manager",
+    "msk": "kafka",
+    "networkfirewall": "network-firewall",
+    "nimblestudio": "nimble",
+    "opensearchservice": "es",
+    "resourcegroups": "resource-groups",
+    "redshiftserverless": "redshift-serverless",
+    "route53recoverycontrol": "route53-recovery-control-config",
+    "route53recoveryreadiness": "route53-recovery-readiness",
+    "s3objectlambda": "s3control",
+    "servicecatalogappregistry": "servicecatalog-appregistry",
+    "ssmcontacts": "ssm-contacts",
+    "ssmincidents": "ssm-incidents",
+    "vpclattice": "vpc-lattice",
+    "wafregional": "waf-regional",
+    "AWS::Timestream::Database": "timestream-write",
+    "AWS::Timestream::ScheduledQuery": "timestream-query",
+    "AWS::Timestream::Table": "timestream-write",
+}
+
+
+def build_index(data_dir):
+    index_path = data_dir / "index.json"
+    index_data = {"resources": {}, "augment": {}}
+    all_services = fake_session().get_available_services()
+
+    for path in sorted(data_dir.rglob("*.json")):
+        if path.name == "index.json":
+            continue
+
+        rdata = json.loads(path.read_text())
+
+        service = path.stem.split("_")[1]
+
+        if service not in all_services:
+            boto_service = ServiceMap.get(service)
+            if boto_service is None:
+                boto_service = ServiceMap.get(rdata["typeName"])
+        else:
+            boto_service = service
+        if not boto_service:
+            print("service not found %s %s" % (rdata["typeName"], service))
+            continue
+
+        raugment = index_data["augment"].setdefault(rdata["typeName"], {})
+        raugment["service"] = boto_service
+
+        rname = path.stem.split("_", 1)[-1]
+        raugment["type"] = rname
+        class_name = "".join([s.title() for s in path.stem.split("_")[1:]])
+        index_data["resources"]["awscc.%s" % rname] = "c7n_awscc.resources.%s.%s" % (
+            path.stem.split("_", 1)[-1],
+            class_name,
+        )
+
+    index_path.write_text(json.dumps(index_data, indent=2))
+
+
+def build(setup_kwargs):
+    data_dir = Path("c7n_awscc") / "data"
```

Comment:
When I tried this locally `data` didn't exist (because this PR empties it), so the script initially failed. May want to guard against that case.

```suggestion
    data_dir = Path("c7n_awscc") / "data"
    data_dir.mkdir(exist_ok=True)
```